MOxUnit/moxunit_runtests.m

```diff
@@ -126,6 +130,18 @@
             mocov_params{k*2}=params.(key);
         end
 
+        % Now do all the keys which map to cell arrays. These are for
+        % arguments which occur multiple times. For these, we make a
+        % parameter pair input for every element in the cell.
+        keys=all_keys(msk_cover & msk_iscell);
+        for k=1:numel(keys)
+            key=keys{k};
+            for c=1:numel(params.(key))
+                mocov_params{end+1}=['-' key];
+                mocov_params{end+1}=params.(key){c};
+            end
+        end
```

Comment:
That looks good to me!
I was trying to avoid the duplication, but original code did not do quite what I thought it did and it broke. This way is a lot cleaner.